mail_debrand/views/mail_notification_view.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
```

Comment:
This is not going to work being noupdate="1".

Reply:
@pedrobaeza What do you mean it won't update a secondary module? Its a completely new record. Just inheriting from an old one, just like any view inheritance, it worked absolutely fine in test.

The old python hack was required because mail.templates weren't written in QWeb and didn't support inheritance. Given QWeb views support inheritance and modification, it would seem to me the correct way to do this.

Reply:
Oh btw, I'm happy to generate the readme, but I tried a few times before making the PR. It isn't really documented and didn't seem to work.  Wants repos and branches, I'm just developing on laptop.

Reply:
Sorry, you're right about that. Indeed, previously this template wasn't Qweb and I have mixed both. Any way, please do several things:

* Are the original records noupdate="1" also? If not, you should do these ones noupdate="'0" for being consistent in the inheritance chain.
* Please put a low priority in the templates, as you are doing `replace` operations.

Reply:
@pedrobaeza 

yes the originals are no update="1". 
I'm not sure how priority works in QWeb, I made it 99, hopefully that is low priority.

Everything else done now as well.